bukkit/src/main/java/me/william278/husksync/bukkit/util/nms/MinecraftVersionUtils.java

```diff
@@ -0,0 +1,42 @@
+package me.william278.husksync.bukkit.util.nms;
+
+import me.william278.husksync.util.ThrowSupplier;
+import org.bukkit.Bukkit;
+
+public class MinecraftVersionUtils {
+
+    public final static String CRAFTBUKKIT_PACKAGE_PATH = Bukkit.getServer().getClass().getPackage().getName();
+
+    public final static String PACKAGE_VERSION = CRAFTBUKKIT_PACKAGE_PATH.split("\\.")[3];
+    public final static String SERVER_VERSION = Bukkit.getBukkitVersion().split("-")[0];
+
+    public final static String MINECRAFT_PACKAGE = compare("1.17") < 0 ?
+            "net.minecraft.server.".concat(PACKAGE_VERSION) : "net.minecraft.server";
+
+    public static int compare(String version) {
```

Comment:
Just tidy up a bit of the styling here
`if (version == null) return 1;`
`for (int i = 0; i < length; i++) {...`